src/optimize.jl

```diff
@@ -95,13 +95,11 @@ function add_infinite_model_optimizer(opt_model::JuMP.Model,
     if !isa(inf_model.optimizer_constructor, Nothing)
         bridge_constrs = JuMP.bridge_constraints(inf_model)
         JuMP.set_optimizer(opt_model, inf_model.optimizer_constructor,
-                           bridge_constraints = bridge_constrs)
+                           add_bridges =  bridge_constrs)
     end
-    # parse the attributes
-    for attr in MOI.get(JuMP.backend(inf_model).model_cache,
-                        MOI.ListOfOptimizerAttributesSet())
-        value = MOI.get(JuMP.backend(inf_model), attr)
-        MOI.set(opt_model, attr, value)
+    # parse the attributes (this is a hacky workaround)
```

Comment:
Hmm. What's this?

Reply:
Because InfiniteOpt is wrapped around an JuMP model (we call it an optimizer model) we want a JuMP model backend to inherit the optimizer and attributes it has. For example, if the user defines an InfiniteModel that will use Ipopt and a time limit we want the JuMP model backend to use Ipopt with that time limit.  By default, we have the `TranscriptionModel` backend which is a JuMP model (with some stuff in `.ext`) that stores the attribute stuff as normal, but if the user wanted to use a different JuMP model backend, we would like it to inherit the solver attributes.